hazelcast/test/src/compact_test.cpp

```diff
@@ -494,6 +495,9 @@ check_schema_field(const schema& schema,
 
 TEST_F(CompactSerializationTest, test_field_order_fixed_size)
 {
+    std::unordered_map<hazelcast::client::decimal, hazelcast::client::decimal> m(4);
+    m.insert(std::make_pair(hazelcast::client::decimal{1, 2},
+                            hazelcast::client::decimal{3, 4}));
```

Comment:
I think these two lines looks weird in this test, we can add this as a new test.

Reply:
It is just a leftover. I removed them.